src/java/org/apache/cassandra/cql3/Operator.java

```diff
@@ -97,7 +118,15 @@ public String toString()
         @Override
         public boolean isSatisfiedBy(AbstractType<?> type, ByteBuffer leftOperand, ByteBuffer rightOperand)
```

Comment:
This method is used by both `ColumnCondition `and `RowFilter`.
Adding these null checks in the Operators method is OK because we were checking for no null values anyway (return false) in `RowFilter `, where we call this method, and the clustering key cannot be null. Is that the correct understanding?

Reply:
Yes. It is the reasoning.